### ISSUE TITLE
[1.5.2] Target class [config] does not exist.

### ISSUE THREAD

**Issue by rcerljenko** [laravel/octane#673]
- Octane Version: 1.5.2
- Laravel Version: 10.6.0
- PHP Version: 8.1
- Server & Version: Swoole 4.9
- Database Driver & Version: MySQL 8.x

### Description:

Hi,

With the latest octane update our server started to throw errors like this on every request.
We use `database` as our session driver if it means something.

```
{
    "class": "Illuminate\\Contracts\\Container\\BindingResolutionException",
    "message": "Target class [config] does not exist.",
    "code": 0,
    "file": "/var/www/html/moj-posao-cms/vendor/laravel/framework/src/Illuminate/Container/Container.php:916",
    "trace": [
        "/var/www/html/moj-posao-cms/vendor/laravel/framework/src/Illuminate/Container/Container.php:795",
        "/var/www/html/moj-posao-cms/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:933",
        "/var/www/html/moj-posao-cms/vendor/laravel/framework/src/Illuminate/Container/Container.php:731",
        "/var/www/html/moj-posao-cms/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:918",
        "/var/www/html/moj-posao-cms/vendor/laravel/framework/src/Illuminate/Container/Container.php:1456",
        "/var/www/html/moj-posao-cms/vendor/laravel/framework/src/Illuminate/Session/SessionManager.php:256",
        "/var/www/html/moj-posao-cms/vendor/laravel/framework/src/Illuminate/Support/Manager.php:68",
        "/var/www/html/moj-posao-cms/vendor/laravel/octane/src/Listeners/GiveNewApplicationInstanceToDatabaseSessionHandler.php:21",
        "/var/www/html/moj-posao-cms/vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php:441",
        "/var/www/html/moj-posao-cms/vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php:249",
        "/var/www/html/moj-posao-cms/vendor/laravel/octane/src/DispatchesEvents.php:20",
        "/var/www/html/moj-posao-cms/vendor/laravel/octane/src/Worker.php:177",
        "/var/www/html/moj-posao-cms/vendor/laravel/octane/bin/swoole-server:145",
        "/var/www/html/moj-posao-cms/vendor/laravel/octane/bin/swoole-server:170"
    ]
}
```

**Comment by nunomaduro** [comment]
Fixed on `1.5.3`.

**Comment by rcerljenko** [comment]
thx man!
